src/main/resources/assets/architecturecraft/lang/zh_CN.lang

```diff
@@ -1,5 +1,6 @@
 tile.architecturecraft:sawbench.name=建筑师锯台
 tile.architecturecraft:shape.name=构件
+tile.architecturecraft:shapeSE.name=构件(发光)
```

Comment:
If you add any localizable text, please make sure to add English (`en_US`) localization (`cause most players play in English).

Reply:
> If you add any localizable text, please make sure to add English (`en_US`) localization (`cause most players play in English).

Thank you! In fact, this is my first time submitting a PR, and I haven't coded in a long time. I appreciate your suggestions and I will make the necessary modifications when I have the time.